exercises/Dimensionality_Reduction/01_Principal_components_analysis.ipynb

```diff
@@ -30,7 +30,7 @@
     "\n",
     "#### How does PCA work?\n",
     "\n",
-    "PCA related eigendecomposition methods are some of the most fundamental dimensionality reduction tools in data science. Many methods, including tSNE and PHATE, first reduce the data using PCA before performing further operations on the data. \n",
+    "PCA related eigen-decomposition methods are some of the most fundamental dimensionality reduction tools in data science. Many methods, including tSNE and PHATE, first reduce the data using PCA before performing further operations on the data. \n",
```

Comment:
Disagree on this one. Eigendecomposition is most frequently one word according to google ngram 
![image](https://user-images.githubusercontent.com/8499679/80544911-15a25200-8980-11ea-91d5-c013640eb662.png)

Reply:
You are right. I thought eigen-decomposition made more sense as English. I will change it back.

Reply:
Fixed now.